src/parsers/transactions.js

```diff
@@ -86,7 +86,21 @@ export const parseTransactions = (
         remainingTransactions
       )
     : concat(updatedPendingTransactions, parsedNewTransactions);
-  return uniqBy(updatedResults, txn => txn.hash);
+
+  const potentialNftTransaction = appended
+    ? find(parsedNewTransactions, txn => {
+        return (
+          !txn.protocol &&
```

Comment:
is this what you meant by not looking at known ERC20s and looking at tx data instead?

Reply:
yes!